static/locales/en-US.yaml

```diff
@@ -519,7 +512,7 @@ Comments:
   Comments: Comments
   Click to View Comments: Click to View Comments
   Getting comment replies, please wait: Getting comment replies, please wait
-  There are no more comments for this video: There are no more comments for this video
+  There are no more comments for this video: No comments for this video
```

Comment:
The 'more' should stay, as there is a difference between No comments available at all and you have reached the end of comments. So something like 'No more comments available for this video'